BloodManagmentSystem/Persistance/EntityConfigurations/DonorConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BloodManagmentSystem.Core.Models;
+using System.Data.Entity.ModelConfiguration;
+
+namespace BloodManagmentSystem.Persistance.EntityConfigurations
+{
+    public class DonorConfiguration : EntityTypeConfiguration<Donor>
+    {
+        public DonorConfiguration()
+        {
+            Property(d => d.Name)
+                .IsRequired()
+                .HasMaxLength(255);
```

Comment:
Name can have less characters, this would cause problems with displaying it, if someone would pus some really long string